templates/scatter_gather_service/src/scatter_gather_service/resources/ballerina.conf

```diff
@@ -0,0 +1,3 @@
+LISTENER_PORT = 9090
+ENDPOINT_1 = "http://www.mocky.io/v2/5d63747c330000796055a838"
+ENDPOINT_2 = "http://www.mocky.io/v2/5d637486330000278255a839"
```

Comment:
Will be better if a separate service can be used here, since it is not advisable to rely on mocky.io endpoints.